src/Response/ResponseInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace ByJG\ApiTools\Response;
+
+interface ResponseInterface
+{
+    public function getHeaders();
+
+    public function getStatusCode();
+
+    public function getBody();
+}
```

Comment:
That's an interface without any documentation or guarantees describing what you can expect from it. I know you said you'd help documenting it, but it's hard to tell the intention with _nothing_.

Reply:
Where would you expect the documentation to be? thank you

Reply:
I've update the documentation. have a great day. Martin

Reply:
That class was removed from implementation in the PR #50 in favor of the Uri implementation.